Assets/Scripts/GameScene/IPlayerController.cs

```diff
@@ -0,0 +1,20 @@
+﻿public interface IPlayerController
+{
+    public Board GetGameBoard();
+
+    private void HandleMovement()
+    {
+    }
+
+    private void HandleRotation()
+    {
+    }
+
+    private void HandleCollision()
+    {
+    }
+
+    private void HandlePhysics()
+    {
+    }
```

Comment:
why do we need the brackets here? it's just a function declaration inside an interface, remove the brackets...